src/networkpolicy/networkPolicy.go

```diff
@@ -2164,8 +2164,8 @@ func StartNetworkCronJob() {
 		log.Error().Msg(err.Error())
 		return
 	}
-	if cfg.CurrentCfg.Observability {
-		err := NetworkCronJob.AddFunc(cfg.GetCfgSysCronJobTime(), observability.NetworkLogCronJob) // time interval
+	if cfg.GetCfgObservabilityEnable() {
+		err := NetworkCronJob.AddFunc(cfg.GetCfgObservabilityCronJobTime(), observability.NetworkLogCronJob) // time interval
```

Comment:
Why do we need a cron job for saving the network/system logs?
In the case of discovery-engine before, we used cron-job because we had to periodically discover the policies .. but in the casee of observability we do not have that requirement. 
We should have a go routine which perpetually waits on the network/system events and pushes it to the log without any cron settings.